mkdocs-project-dir/docs/index.md

```diff
@@ -2,52 +2,71 @@
 layout: docs
 ---
 
-!!! warning
-    Converting and updating this documentation to the new system is still a work in progress, and pages may change content or organisation as we work on it. Please bear with us while we complete the move.
+<!--- ================================================================== --->
+<!---  @file   index.md                                                  --->
+<!---  @brief                                                            --->
+<!---                                                                    --->
+<!---  @details                                                          --->
+<!---                                                                    --->
+<!---  @author(s)                                                        --->
+<!---  @contributor(s) e.kotsialos@ucl.ac.uk                             --->
+<!---  Copyright (c) 2020, UCL:ISD:RITS, RCAS                            --->
+<!---                                                                    --->
+<!--- ================================================================== --->
 
+!!! warning
+    Converting and updating this documentation to the new system is still a work
+    in progress, and pages may change content or organisation as we work on it.
+    Please bear with us while we complete the move.
 
 # Research Computing Services
 
-This documentation is maintained by the research computing team for the purpose
+This documentation is maintained by the Research Computing Team for the purpose
 of sharing information about our services, including user guides,
 service updates and account request and renewal support.
 
 ## Documentation
 
 General User Information:
 
- - [Getting an Account](Account_Services)
- - [Guide for New Users](New_Users)
- - [Guide for Experienced Users](Experienced_Users)
- - [Planned Outages](Planned_Outages)
+- [Getting an Account](Account_Services)
```

Comment:
These should all be .md links really, so they are checked. (Not a change from you, but should be done).